testing/run_tests.py

```diff
@@ -210,14 +270,18 @@ def RunDartTests(build_dir, filter):
       print "Skipping %s due to filter." % dart_test_file
 
 
-def RunTests(build_dir, filter, run_engine_tests, run_dart_tests, run_benchmarks):
+def RunTests(build_dir, filter, run_engine_tests, run_dart_tests, run_java, run_benchmarks):
   if run_engine_tests:
     RunCCTests(build_dir, filter)
 
   # https://github.com/flutter/flutter/issues/36301
   if run_dart_tests and not IsWindows():
     RunDartTests(build_dir, filter)
 
+  # Can't build Android files on Windows
+  if run_java and not IsWindows():
+    RunJavaTests(build_dir, filter)
```

Comment:
Should a warning be printed? Also, why can't we build Android files on Windows?

Reply:
None of the other cases do so I thought it was better to match the existing pattern. I think we may be silent to avoid spammy logs on Windows bots? Not sure what the exactly blocker on Windows is, but it's documented on the Wiki under [compiling the engine](https://github.com/flutter/flutter/wiki/Compiling-the-engine#compiling-a-host-binary-on-windows).